pkgs/applications/video/kodi/plugins.nix

```diff
@@ -556,4 +556,107 @@ let self = rec {
 
     extraBuildInputs = [ libarchive lzma bzip2 zlib lz4 lzo openssl ];
   };
-}; in self
+
+  # an example of a plugin that won't work when packaged with nix
+  # because other packages will modify this package at runtime...
+  skin-arctic-zephyr-mod = mkKodiPlugin rec {
+    plugin = "skin-artic-zephyr-mod";
+    namespace = "skin.arctic.zephyr.mod";
```

Comment:
Can we generate one name of the other?

Reply:
I agree, we should be doing that. Unfortunately not all addons are named according to this yet... but we can fix that.

Reply:
Maybe default it to this and overwrite it where necessary?